bindings/profilers/wall.cc

```diff
@@ -243,18 +241,16 @@ class SignalHandler {
 #endif
 }  // namespace
 
-LabelSetsByNode WallProfiler::GetLabelSetsByNode(CpuProfile* profile,
-                                                 ContextBuffer& contexts) {
-  LabelSetsByNode labelSetsByNode;
+ContextsByNode WallProfiler::GetContextsByNode(CpuProfile* profile,
+                                               ContextBuffer& contexts) {
+  ContextsByNode contextsByNode;
 
   auto sampleCount = profile->GetSamplesCount();
   if (contexts.empty() || sampleCount == 0) {
-    return labelSetsByNode;
+    return contextsByNode;
   }
 
   auto isolate = Isolate::GetCurrent();
-  // auto labelKey = Nan::New<String>("label").ToLocalChecked();
```

Comment:
Ouch, I might've left this one in… thanks for catching it

Reply:
That's might have been me when debugging some issue ;)

Reply:
Funny, I swear I had the same debugging line some weeks ago 😂